content/en/blog/_posts/2025-05-07-kubernetes-1-33-wi-for-image-pulls.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: blog
-title: "From Secrets to Service Accounts: Kubernetes Image Pulls Evolved"
-date: 2025-04-23
+title: "Kubernetes v1.33: From Secrets to Service Accounts: Kubernetes Image Pulls Evolved"
+date: 2025-05-07T10:30:00-08:00
 draft: true
```

Comment:
Sorry forgot to catch this, we need to remove this line so that this gets scheduled for publication

```suggestion
```

Reply:
done!